### PR TITLE
Makozlo/combine create and add commands

### PR DESCRIPTION
**Pull request by makozlo**
- remove the `create` command and adds its functionality to the `add` command
- update the help documentation

### CODE REVIEW

**Review by jonesaustindev — APPROVED** [review]
Nice! I really like this update to the add command

File: src/constants.ts [hunk 1]
```diff
@@ -73,7 +74,70 @@ export const questionTakenFileName = {
...
+  type: 'input',
+  name: 'name',
+  message:
+    'When naming a hook, make sure the hook\'s name follows the pattern useHookName (camelcase starting with "use"). What would you like to name your hook?',
+};
+
+// ------------------------
+// ----- DESCRIPTIONS -----
+// ------------------------
+export const HELP_DESCRIPTION_CMD_ADD = `airfoil add [adr | appcenter | component | env | hook | keystore ]
```
Comment (townofdon) [inline src/constants.ts:87]: Help descriptions already exist in `./src/help-descriptions` - could we move the additional one there?

**Review by townofdon — CHANGES_REQUESTED** [review]
One small change for the help descriptions.

Additionally, I personally think there should be more separation between `install` type commands and `generate` type commands.

`yarn add <package_name>` is already a pattern most devs are familiar with - so `airfoil add <service/package/etc>` would make sense to follow the same pattern.

`airfoil gen` makes sense to me for the following:

- generate ADR
- generate ENV
- generate component / hook
- generate AppIcon
- generate svgs

We prob don't want to change things drastically this close to the finish line. And we can always tweak later, but I have a suspicion that whatever pattern gets used for V1 will be very difficult to change, after people have started adopting the tool.

Just my thoughts! Let me know what you think.

File: src/constants.ts (hunk 1 above)
Comment (townofdon) [reply src/constants.ts:87]: See: https://github.com/teamairship/airfoil/tree/develop/src/help-descriptions

**Pull request closed without merging**